content/en/css.md

```diff
@@ -74,20 +78,24 @@ CSS is most efficient when its selectors are [extremely specific with limited DO
 
 The descendant selector (the space character) is the most expensive selector in CSS. The child selector (the &quot;`>`&quot; character) is also expensive, especially when the rules are tag names rather than classes or IDs. Avoid both. Try applying a class to the element you want to target instead.
 
-```css
+```
```

Comment:
Prism needs "markup" or "css" next to the backticks so the correct CSS class is put on the code block, so the syntax highlighting will be appropriate to the language.